posthog/api/insight.py

```diff
@@ -423,9 +483,13 @@ def get_queryset(self) -> QuerySet:
                     id__in=DashboardTile.objects.exclude(deleted=True).values_list("dashboard_id", flat=True)
                 ),
             ),
-            "dashboards__created_by",
-            "dashboards__team",
             "dashboards__team__organization",
```

Comment:
Shouldn't this be gone too if we want to stop using raw Django `dashboards`?

Reply:
Ultimately yes, but at the moment it's keeping the number of queries down